workers/unity/Assets/Playground/Scripts/Camera/InitCameraSystem.cs

```diff
@@ -5,6 +5,7 @@
 namespace Playground
 {
     [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    [DisableAutoCreation]
```

Comment:
is this the fix for the issue of the reticle not showing up?

Reply:
No, this changes nothing but is required for making it not show up on the default world